### PR TITLE
Create devcontainer.json

### PR DESCRIPTION
**Pull request by SBGiz**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "image": "mcr.microsoft.com/devcontainers/universal:2",
```
Comment (Byurdyzo) [inline .devcontainer/devcontainer.json:2]: Dir/s

**Review by Byurdyzo — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (Byurdyzo) [reply .devcontainer/devcontainer.json:2]: Going to sleep

**Review by Byurdyzo — APPROVED** [review]

**Review by Byurdyzo — APPROVED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (Byurdyzo) [reply .devcontainer/devcontainer.json:2]: Go to shell command

**Review by Byurdyzo — APPROVED** [review]

**Review by Sittisak1991 — COMMENTED** [review]
H

**Review by Sittisak1991 — COMMENTED** [review]
g

**Pull request closed without merging**